app/src/main/java/com/sopt/gongbaek/presentation/ui/component/chip/GroupInfoChip.kt

```diff
@@ -0,0 +1,86 @@
+package com.sopt.gongbaek.presentation.ui.component.chip
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sopt.gongbaek.presentation.type.GroupInfoChipType
+import com.sopt.gongbaek.ui.theme.GongBaekTheme
+
+@Composable
+fun GroupInfoChip(
+    groupInfoChipType: GroupInfoChipType,
+    modifier: Modifier = Modifier
+) {
+    Text(
+        text = stringResource(groupInfoChipType.label),
+        color = groupInfoChipType.fontColor,
+        modifier = modifier
+            .clip(RoundedCornerShape(2.dp))
+            .background(color = groupInfoChipType.backgroundColor)
```

Comment:
[3]: 단순 참고용인데, 이런 방법도 있습니다!!(수정x 참고o)
```suggestion
            .background(
                color = groupInfoChipType.backgroundColor,
                shape = RoundedCornerShape(2.dp)
            )
```

Reply:
이게 더 좋은 방법같아 수정했습니다 😀 👍